front/app/utils/permissions/permissions.ts

```diff
@@ -83,15 +83,15 @@ const hasPermission = ({
 }) => {
   return authUserStream.pipe(
     map((user) => {
-      if (!item) {
+      if (!item || !user) {
         return false;
       }
 
       const resourceType = isResource(item) ? item.type : item;
       const rule = getPermissionRule(resourceType, action);
 
       if (rule && appConfiguration) {
-        return rule(item, user || null, appConfiguration.data, context);
+        return rule(item, user, appConfiguration.data, context);
```

Comment:
Before we would return `null` when `user` was not defined and the code returned `false` in `isProjectModerator` eventually. Just returning `false` earlier in the loop now (when `user` is not defined).